File: CobongMemo/app/src/main/java/cobong/jeongwoojin/cobongmemo/cobongmemo/view/MemoNavigator.java
package cobong.jeongwoojin.cobongmemo.cobongmemo.view;

import cobong.jeongwoojin.cobongmemo.cobongmemo.model.MemoListItem;

public interface MemoNavigator {

    default void sendMemo(MemoListItem item){};
    default void deleteMemo(MemoListItem memo){};
    default void editMemo(MemoListItem memo){};
}

File: CobongMemo/app/src/main/java/cobong/jeongwoojin/cobongmemo/cobongmemo/view/MemoNavigator.kt
package cobong.jeongwoojin.cobongmemo.cobongmemo.view

import cobong.jeongwoojin.cobongmemo.cobongmemo.model.MemoItem

interface MemoNavigator {
    fun sendMemo(item: MemoItem) {}
    fun deleteMemo(memo: MemoItem) {}
    fun editMemo(memo: MemoItem) {}
}
